### PR TITLE
[hotfix] Fix broken link and wrong node version command in runtime-we…

### PR DESCRIPTION
**Pull request by libenchao**
…b readme

## What is the purpose of the change

Fix runtime-web readme broken link for node install instructions.
Fix node version command.

## Brief change log

Fix runtime-web readme broken link for node install instructions.
Fix node version command.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by zentol** [comment]
ping @rmetzger ; aren't these instructions so out-dated that they can be removed?

**Comment by rmetzger** [comment]
@vthinkxie has updated them when the web UI was reworked: https://github.com/apache/flink/commit/dcd92958c58cbf100817e60e8c76ae31ce559b8a#diff-f8eb5f0624e0cf6aa3ce2df626e816c8
So I assume they are fairly up to date.

@vthinkxie what do you think about this PR?

File: flink-runtime-web/README.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ Depending on your version of Linux, Windows or MacOS, you may need to manually i
 
 #### Ubuntu Linux
 
-Install *node.js* by following [these instructions](https://github.com/joyent/node/wiki/installing-node.js-via-package-manager).
+Install *node.js* by following [these instructions](https://github.com/nodesource/distributions/blob/master/README.md#debinstall).
```
Comment (vthinkxie) [inline flink-runtime-web/README.md]:
```suggestion
Install *node.js* by following [these instructions](https://nodejs.org/en/download/).
```

**Comment by vthinkxie** [comment]
Hi @rmetzger 
I didn't update this part last time, left some comments here

**Comment by libenchao** [comment]
@vthinkxie I have addressed your comments, thanks for the review.

**Comment by zentol** [comment]
@rmetzger I thought the whole npm stuff is handled by the frontend-maven-plugin? I can build the WebUI just fine without having npm or node installed.

**Comment by zentol** [comment]
Urom what I see the `Ubuntu Linux / MacOS` should be completely removed, and the `Building/Developing` sections amended to call `mvn package` in `flink-runtime-web` instead.

**Comment by rmetzger** [comment]
@zentol In my understanding, the instructions in the readme are mostly for setting up a local development environment (instead of running mvn for every change on the frontend)
Usually, these frontend frameworks can hot-reload changes etc. (much nicer than running mvn all the time :) )

I agree that the readme does not really reflect that.

**Comment by rmetzger** [comment]
@zentol the issues we've discussed seem orthogonal to this change. Are you ok merging this change to fix the broken URL and command in the readme?

**Comment by zentol** [comment]
@rmetzger yes

**Comment by rmetzger** [comment]
Merging ...

**Pull request closed without merging**